### PR TITLE
Modification in .github/ force all tests in CI

### PR DESCRIPTION
**Pull request by touilleMan**

### CODE REVIEW

File: .github/actions/paths-filter/filters.yml [hunk 1]
```diff
@@ -1,15 +1,18 @@
...
   - parsec/**
   - tests/**
   - pyproject.toml
   - poetry.lock
   - build.py
 rust-ext:
+  - .github/**
   - src/**/*.rs
   - Cargo.toml
   - Cargo.lock
```
Comment (FirelightFlagboy) [inline .github/actions/paths-filter/filters.yml:18]:
```suggestion
rust:
  - oxidation/**/*.rs
  - oxidation/**/*.toml
  - Cargo.toml
  - Cargo.lock
python:
  - parsec/**
  - tests/**
  - pyproject.toml
  - poetry.lock
  - build.py
rust-ext:
  - src/**/*.rs
  - Cargo.toml
  - Cargo.lock
ci:
  - .github/actions/**
  - .github/workflows/**
```
Add another field instead

**Review by FirelightFlagboy — CHANGES_REQUESTED** [review]

File: .github/actions/paths-filter/filters.yml (hunk 1 above)
Comment (touilleMan) [reply .github/actions/paths-filter/filters.yml:18]: I'd rather stay KISS here (on top of that, listing subfolders on `.github/` means we will have issues if we add something new in the `.github/`directly)

Comment (FirelightFlagboy) [reply .github/actions/paths-filter/filters.yml:18]: maybe, listening to the subfolder on `.github` is to much.
But I still think adding the field `ci` is better

Comment (touilleMan) [reply .github/actions/paths-filter/filters.yml:18]: >  But I still think adding the field ci is better

I don't see a direct benefit (except from the "we named it "rust" and it contains ci stuff which is not technically Rust code", hence KISS)

**Review by FirelightFlagboy — APPROVED** [review]

**Pull request merged**